integration_tests/ibc_utils.py

```diff
@@ -171,22 +180,12 @@ def prepare_network(
 
     port = None
     if is_relay:
+        cronos.supervisorctl("start", "relayer-demo")
         if is_hermes:
-            cronos.supervisorctl("start", "relayer-demo")
             port = hermes.port
         else:
-            proc = subprocess.Popen(
-                [
-                    "rly",
-                    "start",
-                    "chainmain-cronos",
-                    "--home",
-                    str(path),
-                ],
-                preexec_fn=os.setsid,
-            )
             port = 5183
-    yield IBCNetwork(cronos, chainmain, hermes, incentivized, proc)
+    yield IBCNetwork(cronos, chainmain, hermes, incentivized)
```

Comment:
why not using the `network.proc`?

Reply:
to return this [proc](https://github.com/crypto-org-chain/cronos/blob/f74dd4959ddb384145b7bbfaefefc3ad7001ecb3/integration_tests/ibc_utils.py#L148) to ensure process can be killed even early crash

Reply:
https://github.com/crypto-org-chain/cronos/pull/1200
this should solve the exception handling issue.

Reply:
yes 🪄